### PR TITLE
Fix cumulus companion checks

### PR DESCRIPTION
**Pull request by bkchr**
The fix is here https://github.com/paritytech/pipeline-scripts/pull/12

### CODE REVIEW

**Review by thiolliere — APPROVED** [review]
seems good to me

**Review by shawntabrizi — APPROVED** [review]

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -606,6 +607,7 @@ cargo-check-macos:
         --substrate
         "$DEPENDENT_REPO"
         "$GITHUB_PR_TOKEN"
+        $CARGO_UPDATE_CRATES
```
Comment (TriplEight) [inline .gitlab-ci.yml:610]:
```suggestion
        "$CARGO_UPDATE_CRATES"
```

**Review by TriplEight — DISMISSED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (bkchr) [reply .gitlab-ci.yml:610]: I tested this and this is wrong.

**Comment by bkchr** [comment]
bot merge

**Review by TriplEight — APPROVED** [review]

**Pull request merged**